Bookish.Web/Controllers/HomeController.cs

```diff
@@ -47,6 +48,30 @@ public IActionResult Copies(string isbn)
             return View(model);
         }
 
+        public IActionResult NewBook()
```

Comment:
Is this the get endpoint which goes with the post endpoint below? In which case I'd expect it to be called AddBook as well (if the two endpoints end up taking the same arguments then you can name the post one `AddBookPost` and use the action name attribute to set the action names to still be ther same: https://stackoverflow.com/questions/9552761/get-and-post-methods-with-the-same-action-name-in-the-same-controller). I would also expect this to take the previously entered values as arguments, so that if the user is redirected here because there was an error with their input then they can see what they previously entered 🙂